README.md

```diff
@@ -164,40 +204,83 @@ import { Klaviyo } from 'klaviyo-react-native-sdk';
+- [iOS](https://github.com/klaviyo/klaviyo-swift-sdk#Push-Notifications)
 
-## Deep Linking
+### Setup
+Refer to the following README sections on push setup:
+- [Android](https://github.com/klaviyo/klaviyo-android-sdk#Setup)
+- [iOS](https://github.com/klaviyo/klaviyo-swift-sdk#Push-Notifications
 
-To handle deep links in your app, start by familiarizing yourself with the React Native [guide](https://reactnative.dev/docs/linking) to deep linking. Once you've done that, you should follow directions from the [iOS](https://github.com/klaviyo/klaviyo-swift-sdk?tab=readme-ov-file#Deep-Linking) and [Android](https://github.com/klaviyo/klaviyo-android-sdk?tab=readme-ov-file#Deep-Linking) SDKs.
-The sections below give additional details for each platform as it pertains to React Native.
+[//]: # (TODO - change swift-sdk link to #Setup)
 
-### iOS
+### Collecting Push Tokens
+Push tokens must be collected in the native layer. Follow the platform-specific instructions below:
+- [Android](https://github.com/klaviyo/klaviyo-android-sdk#Collecting-Push-Tokens)
+- [iOS](https://github.com/klaviyo/klaviyo-swift-sdk#sending-push-notifications)
 
-As shown in the native SDK documentation, you can follow option 1 or 2.
+[//]: # (TODO - change swift-sdk link to #Collecting-Push-Tokens)
 
-With option 1, when you get the callback, you can handle it as follows:
+#### Notification Permission
+Requesting user permission to display notifications can be managed in the native layer as instructed in our native SDK
+documentation, or with a third party library that provides cross-platform permissions APIs. If you opt for a
+cross-platform permission solution, you will still need to provide the Klaviyo SDK with the push token from the
+native layer after a permission change.
 
-```objective-c
-[RCTLinkingManager application:application openURL:url options:options]
-```
+### Receiving Push Notifications
+You can send test notifications to a specific token using the
+[push notification preview](https://help.klaviyo.com/hc/en-us/articles/18011985278875)
+feature in order to test your integration.
 
-Since you won't have `options`, you can just pass in an empty dictionary for that parameter.
+#### Rich Push
+[Rich Push](https://help.klaviyo.com/hc/en-us/articles/16917302437275) is the ability to add images to
+push notification messages. On iOS, you will need to implement an extension service to attach images to notifications.
+No additional setup is needed to support rich push on Android.
+- [Android](https://github.com/klaviyo/klaviyo-android-sdk#Rich-Push)
+- [iOS](https://github.com/klaviyo/klaviyo-swift-sdk#rich-push-notifications)
 
-With option 2, when you handle the open url (in [`application(_:open:options)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application)),
-you call the linking code block above similar to what you would do with option 1.
+[//]: # (TODO - change swift-sdk link to #Rich-Push)
```

Comment:
we could just make the url what we think itll be and itll just fallback to the repo page

Reply:
and fast follow with the iOS read update, either way at some point the links will be out of sync

Reply:
so doesnt really matter, just figured this would mean one less PR

Reply:
That was what i did in my first draft, but I think it'd be best to have working links so we can publish this tomorrow and not worry about the iOS draft if it takes till monday or something

Reply:
I mean i know they won't 404 if the header doesn't exist, but still best to have them point to the relevant spot.

Reply:
I put the expected links back, since it looks like Ajay will be done the iOs readme soon and we won't release this till monday anyway.